sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosBulkOperationRequestAndResponse.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import com.azure.cosmos.util.Beta;
+import reactor.core.publisher.Flux;
+
+/**
+ * Request, response and the exception(if any) for a {@link CosmosItemOperation} request when processed using Bulk by calling
+ * {@link CosmosAsyncContainer#processBulkOperations(Flux, BulkProcessingOptions)}.
+ */
+@Beta(Beta.SinceVersion.V4_7_0)
+public final class CosmosBulkOperationRequestAndResponse<TContext> {
```

Comment:
IMO CosmosBulkOperationResposne is sufficient - like in HttpClient's API HttpResponseMessage also has a link back to request.

Reply:
Fixed. Thanks.